include/constructors/constructors.hpp

```diff
@@ -39,7 +39,15 @@ BigInt::BigInt(const BigInt& num) {
 */
 
 BigInt::BigInt(const long long& num) {
-    magnitude = { (unsigned long long) llabs(num) };
+    if (sizeof(long long) == (sizeof(uint64_t)))
```

Comment:
Instead of these checks at multiple places, I suggest we change all occurrences of `long long` to `int64_t` and `unsigned long long` to `uint64_t`.